cobra/core/model.py

```diff
@@ -967,6 +967,34 @@ def objective(self, value):
             value = {rxn: 1 for rxn in reactions}
         set_objective(self, value, additive=False)
 
+    @property
+    def objective_direction(self):
+        """
+        Get or set the objective direction.
+
+        When using a `HistoryManager` context, this attribute can be set
+        temporarily, reversed when exiting the context.
+
+        """
+        return self.solver.objective.direction
+
+    @objective_direction.setter
+    def objective_direction(self, value):
```

Comment:
would this work with the simple `@resettable` decorator?

Reply:
Good call. It should indeed work.